apps/loan/src/components/PageCrvUsdStaking/UserInformation/index.tsx

```diff
@@ -0,0 +1,130 @@
+import styled from 'styled-components'
+import { t } from '@lingui/macro'
+import Image from 'next/image'
+
+import { RCCrvUSDLogoSM, RCScrvUSDLogoSM, RCYieldGrowth } from 'ui/src/images'
+
+import Icon from '@/ui/Icon'
+import { ExternalLink } from '@/ui/Link'
+
+type UserInformationProps = {
+  className?: string
+}
+
+const UserInformation = ({ className }: UserInformationProps) => {
+  return (
+    <UserInformationWrapper className={className}>
+      <Title>{t`How to use the crvUSD Savings vault?`}</Title>
+      <TextContainerRow>
+        <InformationContainer>
+          <Image src={RCCrvUSDLogoSM} alt="crvUSD logo" width={48} height={48} />
+          <InformationTitle>{t`Get crvUSD`}</InformationTitle>
+          <InformationParagraph>{t`To access the yield of the Curve Savings Vault, you need to acquire crvUSD on the open markets or borrow it in the LLAMALEND markets.`}</InformationParagraph>
+          <InformationParagraph>
+            {t`We recommend using Curve's`}{' '}
+            <StyledExternalLinkInline href="https://curve.fi/#/ethereum/swap">QuickSwap</StyledExternalLinkInline>
```

Comment:
Let's use APP_LINK so it works everywhere: https://github.com/curvefi/curve-frontend/blob/d05af9eb268ec081a1bf5036b37c7f81833dbe1c/packages/ui/src/AppNav/index.ts#L10